### PR TITLE
[CM_720] | Dheeraj/Bipul | Add API contracts for forgot consent pin

### PR DESCRIPTION
**Pull request by dheeraj-p**

### CODE REVIEW

File: src/main/resources/static/api.yaml [hunk 1]
```diff
@@ -1561,6 +1561,103 @@ paths:
...
+              * Downstream services are down
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
+  /patients/forgot-pin/update-pin:
```
Comment (ganesanarun) [inline src/main/resources/static/api.yaml:1625]: there are only two things right, one I know my existing pin and try to update the pin, or reset-pin.

URL could be as simple as /patients/reset-pin

File: src/main/resources/static/api.yaml [hunk 2]
```diff
@@ -1561,6 +1561,103 @@ paths:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
 
+  /patients/forgot-pin/generate-otp:
+    post:
+      tags:
+        - account
+      summary: Generate OTP for forgot pin flow
+      parameters:
```
Comment (ganesanarun) [inline src/main/resources/static/api.yaml:1569]: It's fine now, we should try to unify the all OTP based thing into one single API, where generate\_OTP should take the on what scope user is trying to generate OTP.

File: src/main/resources/static/api.yaml (hunk 1 above)
Comment (dheeraj-p) [reply src/main/resources/static/api.yaml:1625]: Yes, I can be. We will change it.

File: src/main/resources/static/api.yaml (hunk 2 above)
Comment (dheeraj-p) [reply src/main/resources/static/api.yaml:1569]: Yes, same thing we were thinking, We will create a tech debt card for that.

**Pull request merged**